test/runtests.jl

```diff
@@ -19,11 +20,12 @@ validate_c(c) = begin
                 end
 
 validate_e(e) = begin
+                    T = TypeVar(:T, AbstractString)
```

Comment:
I don't think it is. That's an unrelated change that accidentally made it in. I'll take it out.